### ISSUE TITLE
multiple-level attribute imports

### ISSUE THREAD

**Issue by rocky** [andrew-tavera/unpyc37#3]
compiling and decompiling: 

```
import anothermodule.stuff.stuff2.stuff3 as stuff3
```
gives

```
Traceback (most recent call last):
  File "unpyc3.py", line 2893, in <module>
    print(decompile(sys.argv[1]))
  File "unpyc3.py", line 164, in decompile
    return dec_module(obj)
  File "unpyc3.py", line 154, in dec_module
    return code.get_suite(include_declarations=False, look_for_docstring=True)
  File "unpyc3.py", line 433, in get_suite
    dec.run()
  File "unpyc3.py", line 1597, in run
    new_addr = method(addr, arg)
  File "unpyc3.py", line 1997, in STORE_NAME
    self.store(name)
  File "unpyc3.py", line 1620, in store
    val.store(self, dest)
  File "unpyc3.py", line 1250, in store
    assert isinstance(imp, ImportStatement)
```

https://github.com/rocky/python-uncompyle6/tree/master/test/bytecode_3.7 and 
https://github.com/rocky/python-uncompyle6/tree/master/test/bytecode_3.7_run 

have some interesting test cases to run to look for 3.7 compilation problems.

**Comment by andrew-tavera** [comment]
Thanks again rocky, should be fixed in 0e57a70d598237802bd35e68e6685e70fefe97af

**Comment by rocky** [comment]
Great!
